.Rbuildignore

```diff
@@ -23,3 +23,5 @@
 ^codecov\.yml$
 ^Rmarkdown$
 ^\.lintr$
+^_targets\.R$
+^_targets$
```

Comment:
I think only one of these is needed?

Reply:
The _targets.R file sits outside of the folder. However i am not convinced i have done this right as the `_targets.R` file still shows in my console when i switch branch - i committed this to mainR so i would expect it to disappear?

Reply:
[`.Rbuildignore`](https://github.com/Public-Health-Scotland/source-linkage-files/pull/544/files/84c4e5129c6989d01864ec9b7cda03e78b2fabc3..bfe04c38888aa041be0ebf9b3941a98e9cf621b2#diff-14fda507bd771db16c0fcb3a7d99002e54911cbe9f86bb2a6002562ee0fc0927) is for getting R CMD CHECK to ignore files but `.gitignore` for getting git to ignore files.

We want `_targets.R` to be tracked by git but not for R CMD to complain about it.

If you commit all the changes to `_targets.R` before you change branches, it should then just disappear. It's just that the uncommitted changes are transferring with you when you change branches.

For the `.Rbuildignore` above `^_targets\.R$` will find a file called (exactly) `_targets.R` whereas `^_targets$` will find a file called (exactly) `_targets` so I think you want the first line only?

Reply:
Thanks @Moohan this is really useful to know the difference and the reason why the file keeps appearing! I will change this to find `_targets\.R` only as i had thought this would also need to pick up the targets folder which is why i also included `^_targets$`. But good to know that this is not needed in this instance and the `.Rbuildignore` is looking for specific files.